lisa/tools/lscpu.py

```diff
@@ -54,7 +54,9 @@ class Lscpu(Tool):
     __vcpu = re.compile(r"^CPU\(s\):[ ]+([\d]+)\r?$", re.M)
     __thread_per_core = re.compile(r"^Thread\(s\) per core:[ ]+([\d]+)\r?$", re.M)
     __core_per_socket = re.compile(r"^Core\(s\) per socket:[ ]+([\d]+)\r?$", re.M)
+    __core_per_cluster = re.compile(r"^Core\(s\) per cluster:[ ]+([\d]+)\r?$", re.M)
```

Comment:
Please add examples above, what data is captured.

Reply:
Done